src/components/manga/TrackMangaButton.tsx

```diff
@@ -28,8 +29,67 @@ export const TrackMangaButton = ({ manga }: { manga: TManga }) => {
 
     const loggedInTrackers = Trackers.getLoggedIn(trackerList.data?.trackers.nodes ?? []);
     const trackersInUse = Trackers.getLoggedIn(Trackers.getTrackers(mangaTrackers));
+    const mangaChaptersQuery = requestManager.useGetMangaChapters(manga.id, {});
```

Comment:
I this the line you are talking about that is fetching the chapters? @schroda

Reply:
no, it's this

```
        // Fetch new chapters if behind tracker
        if (localBehindTracker) {
            await requestManager.getMangaChaptersFetch(manga.id, { awaitRefetchQueries: true }).response;
        }
```

Reply:
Ok. If the local manga is behind (last chapter is 100 but last read in tracker is 103) there wont be a way to find the last chapter that match the tracker. In that case, do you propose to not get the last chapter and instead mark every as read?. Since in the example, the last local chapter is 100 and the last tracker read is 103, every before 103 should be read, in this case all the local chapters should be marked as read?

Reply:
yes, I am

Reply:
Ok. I can do that.